src/notification/index.js

```diff
@@ -77,6 +78,10 @@ export const getAccountFromNotificationData = (
       realm_uri,
       parsed_url: realmUrl,
       match_count: urlMatches.length,
+      known_urls: knownUrls,
+      uniqueIdentities: identities
+        .map(identity => `${identity.realm.origin} ${identity.email}`)
+        .filter((value, index, self) => self.indexOf(value) === index).length,
```

Comment:
This should probably count only the ones that were in the list of matches, right? Otherwise it gets some noise that makes it harder to interpret -- we never know for sure that there wasn't a dupe, because it could just represent some unrelated accounts in other orgs.

You can use `urlMatches` to reuse that computation without duplicating it -- it's a list of indexes.